### PR TITLE
Wordsmith pattern exhaustiveness

### PR DESCRIPTION
**Pull request by Nigel-Ecma**
Addresses #939

### CODE REVIEW

File: standard/patterns.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ A set of patterns `Q` *subsumes* a pattern `P` if any of the following condition
 
 ## 11.4 Pattern exhaustiveness
 
-Informally, a set of patterns is exhaustive for a type if some pattern in the set is applicable to every possible value of that type other than null.
+Informally, a set of patterns is exhaustive for a type if for every possible value of that type other than null some pattern in the set is applicable.
```
Comment (jskeet) [inline standard/patterns.md]:
```suggestion
Informally, a set of patterns is exhaustive for a type if, for every possible value of that type other than null, some pattern in the set is applicable.
```

Personally I think the commas help the reader pause in the right place - they're not *needed*, but I think they help. See what you think.

**Review by jskeet — APPROVED** [review]
Generally happy - one suggestion.

**Review by RexJaeschke — APPROVED** [review]

File: standard/patterns.md (hunk 1 above)
Comment (Nigel-Ecma) [reply standard/patterns.md]: I consulted a statistically significant sample size of one ;-) and they agreed that they not needed but if adding any they would add your second one and not the first. I'll admit to just following the comma count in the original! Let's see if @RexJaeschke proposes a change.

**Review by BillWagner — APPROVED** [review]
LGTM

**Pull request merged**